### DISCUSSION TITLE
not work without class="language-xxxxx"

### DISCUSSION THREAD

**Discussion by ghost** [PrismJS/prism#2969] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6580897/123500603-db127300-d671-11eb-8991-b25f5795afcc.png)

i hope it like github  :  
  when i submit  ``` `a=123` ```

then, show like this 
![image](https://user-images.githubusercontent.com/6580897/123500629-1745d380-d672-11eb-989d-bec6f9b52136.png)

**Comment by RunDevelopment** [comment, marked as answer]
Please read our [Basic usage section](https://prismjs.com/#basic-usage).

The `language-xxxx` class is always necessary. If you want Prism to *style* all code blocks without *highlighting* them, then add a `language-none` or `language-plain` class to the `<body>` of your page. For more information, see the Language inheritance section.

**Reply by ghost** [reply]
thank you very much
